rest/src/main/kotlin/com/gitlab/kordlib/rest/request/HttpUtils.kt

```diff
@@ -50,8 +50,8 @@ fun HttpResponse.logString(body: String) = "[RESPONSE]:${status.value}:${call.re
 
 suspend fun HttpResponse.errorString(): String {
     val message = String(this.readBytes())
-    return if (message.isBlank()) "${logString(message)}"
-    else "${logString(message)}"
+    return if (message.isBlank()) logString(message)
```

Comment:
we can just return the log without checking the message content can't we?

Reply:
yeah, this seems to be the result of simplifying a condition to the point it is no longer needed 😅